Author: Tianon Gravi <admwiggin@gmail.com>
Description: "archive/tar" patch for upstreamed xattrs patch
Applied-Upstream: when golang-1.4 is broadly packaged (scheduled to be released 2014-12-01)

diff --git a/graph/tags_unit_test.go b/graph/tags_unit_test.go
index 58ad8ed..0b06f3e 100644
--- a/graph/tags_unit_test.go
+++ b/graph/tags_unit_test.go
@@ -7,11 +7,11 @@ import (
 	"path"
 	"testing"
 
+	"archive/tar"
 	"github.com/docker/docker/daemon/graphdriver"
 	_ "github.com/docker/docker/daemon/graphdriver/vfs" // import the vfs driver so it is used in the tests
 	"github.com/docker/docker/image"
 	"github.com/docker/docker/utils"
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
 )
 
 const (
diff --git a/integration-cli/docker_api_containers_test.go b/integration-cli/docker_api_containers_test.go
index 5dce388..fca39c4 100644
--- a/integration-cli/docker_api_containers_test.go
+++ b/integration-cli/docker_api_containers_test.go
@@ -11,8 +11,8 @@ import (
 	"testing"
 	"time"
 
+	"archive/tar"
 	"github.com/docker/docker/api/stats"
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
 )
 
 func TestContainerApiGetAll(t *testing.T) {
diff --git a/integration-cli/docker_cli_push_test.go b/integration-cli/docker_cli_push_test.go
index bcab531..2b5aa4b 100644
--- a/integration-cli/docker_cli_push_test.go
+++ b/integration-cli/docker_cli_push_test.go
@@ -9,7 +9,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 // pulling an image from the central registry should work
diff --git a/integration-cli/utils.go b/integration-cli/utils.go
index f67ee78..77c5720 100644
--- a/integration-cli/utils.go
+++ b/integration-cli/utils.go
@@ -15,7 +15,7 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 func getExitCode(err error) (int, error) {
diff --git a/integration/api_test.go b/integration/api_test.go
index ab2c307..a520355 100644
--- a/integration/api_test.go
+++ b/integration/api_test.go
@@ -14,11 +14,11 @@ import (
 	"testing"
 	"time"
 
+	"archive/tar"
 	"github.com/docker/docker/api"
 	"github.com/docker/docker/api/server"
 	"github.com/docker/docker/engine"
 	"github.com/docker/docker/runconfig"
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
 )
 
 func TestSaveImageAndThenLoad(t *testing.T) {
diff --git a/integration/utils_test.go b/integration/utils_test.go
index 32ca8e0..a283597 100644
--- a/integration/utils_test.go
+++ b/integration/utils_test.go
@@ -14,7 +14,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 
 	"github.com/docker/docker/builtins"
 	"github.com/docker/docker/daemon"
diff --git a/pkg/archive/archive.go b/pkg/archive/archive.go
index 68e5c1d..b54bad3 100644
--- a/pkg/archive/archive.go
+++ b/pkg/archive/archive.go
@@ -16,7 +16,7 @@ import (
 	"strings"
 	"syscall"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/docker/pkg/fileutils"
diff --git a/pkg/archive/archive_test.go b/pkg/archive/archive_test.go
index 6cd95d5..dbe8d5e 100644
--- a/pkg/archive/archive_test.go
+++ b/pkg/archive/archive_test.go
@@ -14,7 +14,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 func TestCmdStreamLargeStderr(t *testing.T) {
diff --git a/pkg/archive/archive_unix.go b/pkg/archive/archive_unix.go
index c0e8aee..dd1b767 100644
--- a/pkg/archive/archive_unix.go
+++ b/pkg/archive/archive_unix.go
@@ -6,7 +6,7 @@ import (
 	"errors"
 	"syscall"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 func setHeaderForSpecialDevice(hdr *tar.Header, ta *tarAppender, name string, stat interface{}) (nlink uint32, inode uint64, err error) {
diff --git a/pkg/archive/archive_windows.go b/pkg/archive/archive_windows.go
index 3cc2493..6312df3 100644
--- a/pkg/archive/archive_windows.go
+++ b/pkg/archive/archive_windows.go
@@ -3,7 +3,7 @@
 package archive
 
 import (
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 func setHeaderForSpecialDevice(hdr *tar.Header, ta *tarAppender, name string, stat interface{}) (nlink uint32, inode uint64, err error) {
diff --git a/pkg/archive/changes.go b/pkg/archive/changes.go
index 85217f6..bc67e90 100644
--- a/pkg/archive/changes.go
+++ b/pkg/archive/changes.go
@@ -10,7 +10,7 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/docker/pkg/pools"
diff --git a/pkg/archive/diff.go b/pkg/archive/diff.go
index ca28207..1e9b968 100644
--- a/pkg/archive/diff.go
+++ b/pkg/archive/diff.go
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"syscall"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 
 	"github.com/docker/docker/pkg/pools"
 	"github.com/docker/docker/pkg/system"
diff --git a/pkg/archive/diff_test.go b/pkg/archive/diff_test.go
index 758c411..1af10fe 100644
--- a/pkg/archive/diff_test.go
+++ b/pkg/archive/diff_test.go
@@ -3,7 +3,7 @@ package archive
 import (
 	"testing"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 func TestApplyLayerInvalidFilenames(t *testing.T) {
diff --git a/pkg/archive/utils_test.go b/pkg/archive/utils_test.go
index 9048027..539f694 100644
--- a/pkg/archive/utils_test.go
+++ b/pkg/archive/utils_test.go
@@ -9,7 +9,7 @@ import (
 	"path/filepath"
 	"time"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 var testUntarFns = map[string]func(string, io.Reader) error{
diff --git a/pkg/archive/wrap.go b/pkg/archive/wrap.go
index b8b6019..dfb335c 100644
--- a/pkg/archive/wrap.go
+++ b/pkg/archive/wrap.go
@@ -1,8 +1,8 @@
 package archive
 
 import (
+	"archive/tar"
 	"bytes"
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
 	"io/ioutil"
 )
 
diff --git a/pkg/tarsum/tarsum.go b/pkg/tarsum/tarsum.go
index 88fcbe4..17ca379 100644
--- a/pkg/tarsum/tarsum.go
+++ b/pkg/tarsum/tarsum.go
@@ -12,7 +12,7 @@ import (
 	"io"
 	"strings"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 const (
diff --git a/pkg/tarsum/tarsum_test.go b/pkg/tarsum/tarsum_test.go
index 26f12cc..3ec7035 100644
--- a/pkg/tarsum/tarsum_test.go
+++ b/pkg/tarsum/tarsum_test.go
@@ -15,7 +15,7 @@ import (
 	"os"
 	"testing"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 type testLayer struct {
diff --git a/pkg/tarsum/versioning.go b/pkg/tarsum/versioning.go
index 0ceb529..243844e 100644
--- a/pkg/tarsum/versioning.go
+++ b/pkg/tarsum/versioning.go
@@ -6,7 +6,7 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 )
 
 // versioning of the TarSum algorithm
